packages/ui/header/UserMenu.tsx

```diff
@@ -105,14 +105,14 @@ export const NoUserLink = forwardRef<Omit<LinkProps, 'children'>, 'a'>(
             borderRadius="full"
             display="flex"
             alignItems="center"
+            justifyContent="center"
             {...props}
+            h={8}
+            w={8}
+            _hover={{ opacity: '0.7' }}
+            transition="0.3s"
          >
-            <NoUserIcon
-               transition="color 300ms"
-               _hover={{
-                  color: 'brand.300',
-               }}
-            />
+            <FaIcon icon={faUser} h="5" color="white" />
```

Comment:
Was the `NoUserIcon` the same sort of shape as `faUser`? Only asking because I could picture a `NoUserIcon` that maybe had a slash through it or something (probably not, in this context, but you never know).

Reply:
The new icon has not a circular background anymore. The change in the aspect is like this:
| Old  | New |
| ------------- | ------------- |
| <img width="101" alt="Screenshot 2022-09-22 at 14 58 39" src="https://user-images.githubusercontent.com/7805759/191754238-9a79ae61-0f51-4bd8-9098-1334cdf1c6ad.png"> | <img width="100" alt="Screenshot 2022-09-22 at 14 58 56" src="https://user-images.githubusercontent.com/7805759/191754319-21377de1-77eb-4a20-a788-61767f172dc0.png"> |

I think @mmarcon91 is going to change the mockup to reflect this change because we discussed this previously